Add Babylon.js

Symbitic opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

**https://github.com/Symbitic/awesome-babylonjs**

**The list contains resources about the popular open-source game engine Babylon.js - a framework for web and native game development written in TypeScript.**

> PR's reviewed:
> 
> * [Add Hackathon #2433 (comment)](https://github.com/sindresorhus/awesome/pull/2433#issuecomment-1409775245)
> * [Add Awesome ChatGPT Prompts #2449 (comment)](https://github.com/sindresorhus/awesome/pull/2449#issuecomment-1409781759)

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

Symbitic commented:
@sindresorhus I have filed issues for the three failing items on awesome-lint:

- [False positive: identical links](https://github.com/sindresorhus/awesome-lint/issues/158)
- [Exception for title case and list item punctuation ending with .js](https://github.com/sindresorhus/awesome-lint/issues/157)

Babylon.js playground URLs include hashtags (<https://playground.babylonjs.com/#58I88I#186>), so awesome-lint incorrectly flags them as duplicates.

The other two errors (remark-lint:awesome-heading and remark-lint:awesome-list-item) are caused by the fact that "Babylon.js" has a period in it. Being as this causes other lists like awesome-nodejs to fail, I believe there should be an exception added to when a sentence/title ends with ".js" (or any other extension).

Superlegend717 commented:
Please sir i am a learner ,i need know how can i used termux in android,please!

StealAngleK21 commented on readme.md:
```diff
@@ -556,6 +556,7 @@
 - [Learn Gamedev](https://github.com/notpresident35/awesome-learn-gamedev#readme) - The craft of video game creation.
+- [Babylon.js](https://github.com/Symbitic/awesome-babylonjs#readme) - TypeScript game engine for cross-platform web and native game development.
 
 ## Development Environment
```
# (3256427562)_0r@

kaniaheard971@gmail.com

😉😋😍

StealAngleK21 commented on readme.md:
```diff
@@ -556,6 +556,7 @@
 - [Games of Coding](https://github.com/michelpereira/awesome-games-of-coding#readme) - Learn a programming language by making games.
 - [Esports](https://github.com/strift/awesome-esports#readme) - Video games played as a sport.
 - [Learn Gamedev](https://github.com/notpresident35/awesome-learn-gamedev#readme) - The craft of video game creation.
+- [Babylon.js](https://github.com/Symbitic/awesome-babylonjs#readme) - TypeScript game engine for cross-platform web and native game development.
```
I know

StealAngleK21 reviewed (commented):
I'm perfectly fine... I know now... And I'll be taking my Mt back

zakarialaoui10 commented:
The linter returns 108 errors

✖    1:1   Main heading must be in title case             remark-lint:awesome-heading
  ✖   35:3   https://playground.babylonjs.com/              remark-lint:double-link
  ✖   63:16  https://playground.babylonjs.com/#58I88I#186   remark-lint:double-link
  ✖   64:11  https://playground.babylonjs.com/#947JF6#0     remark-lint:double-link
  ✖   65:23  https://playground.babylonjs.com/#8T3YBR#2     remark-lint:double-link
  ✖   66:34  https://playground.babylonjs.com/#WVPVWL       remark-lint:double-link
  ✖   67:28  https://playground.babylonjs.com/#VXA85M#5     remark-lint:double-link
  ✖   68:25  https://playground.babylonjs.com/#9CMU0F       remark-lint:double-link
  ✖   69:16  https://playground.babylonjs.com/#AL6VAN#1     remark-lint:double-link
  ✖   70:32  https://playground.babylonjs.com/#KBS9I5#925   remark-lint:double-link
  ✖   71:13  https://playground.babylonjs.com/#6QYHV7#29    remark-lint:double-link
  ✖   72:18  https://playground.babylonjs.com/#7S3ZHU#[15](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:16)    remark-lint:double-link
  ✖   73:30  https://playground.babylonjs.com/#CUH660#27    remark-lint:double-link
  ✖   74:33  https://playground.babylonjs.com/#0PHDVB#0     remark-lint:double-link
  ✖   75:26  https://playground.babylonjs.com/#1YD970#283   remark-lint:double-link
  ✖   76:24  https://playground.babylonjs.com/#1YD970#14    remark-lint:double-link
  ✖   77:29  https://playground.babylonjs.com/#502Z8L#0     remark-lint:double-link
  ✖   78:21  https://playground.babylonjs.com/#AH85PF#53    remark-lint:double-link
  ✖   79:17  https://playground.babylonjs.com/#092XX3#13    remark-lint:double-link
  ✖   80:30  https://playground.babylonjs.com/#U8MEB0#114   remark-lint:double-link
  ✖   81:35  https://playground.babylonjs.com/#RGNDK2#2     remark-lint:double-link
  ✖   82:17  https://playground.babylonjs.com/#3FJZD5#4     remark-lint:double-link
  ✖   83:24  https://playground.babylonjs.com/#D7LQDZ#1     remark-lint:double-link
  ✖   84:34  https://playground.babylonjs.com/#355XBJ#[16](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:17)    remark-lint:double-link
  ✖   85:19  https://playground.babylonjs.com/#5MJ1X9#13    remark-lint:double-link
  ✖   86:23  https://playground.babylonjs.com/#5ZCGRM#2860  remark-lint:double-link
  ✖   87:24  https://playground.babylonjs.com/#9U086#4      remark-lint:double-link
  ✖   88:27  https://playground.babylonjs.com/#HDHQN#55     remark-lint:double-link
  ✖   89:26  https://playground.babylonjs.com/#HDHQN        remark-lint:double-link
  ✖   90:43  https://playground.babylonjs.com/#CWAQ1W#12    remark-lint:double-link
  ✖   91:26  https://playground.babylonjs.com/#VS5XS7#0     remark-lint:double-link
  ✖   92:31  https://playground.babylonjs.com/#4HUQQ#1479   remark-lint:double-link
  ✖   93:33  https://playground.babylonjs.com/#RDJ26Q#1288  remark-lint:double-link
  ✖   94:33  https://playground.babylonjs.com/#UL7W2M       remark-lint:double-link
  ✖   95:25  https://playground.babylonjs.com/#22KIIK#22    remark-lint:double-link
  ✖   96:20  https://playground.babylonjs.com/#LVJG7H       remark-lint:double-link
  ✖   97:29  https://playground.babylonjs.com/#45FPFC#13    remark-lint:double-link
  ✖   98:16  https://playground.babylonjs.com/#6MQB2X#158   remark-lint:double-link
  ✖   99:35  https://playground.babylonjs.com/#HY823G#7     remark-lint:double-link
  ✖  100:36  https://playground.babylonjs.com/#7XU4D8#1     remark-lint:double-link
  ✖  101:23  https://playground.babylonjs.com/#ISZ7Y2       remark-lint:double-link
  ✖  102:36  https://playground.babylonjs.com/#HJZBRG#100   remark-lint:double-link
  ✖  103:16  https://playground.babylonjs.com/#E8C51D#35    remark-lint:double-link
  ✖  104:28  https://playground.babylonjs.com/#0IRV8X#2     remark-lint:double-link
  ✖  105:32  https://playground.babylonjs.com/#BP7QYG#13    remark-lint:double-link
  ✖  106:12  https://playground.babylonjs.com/#A8X91Q#10    remark-lint:double-link
  ✖  107:18  https://playground.babylonjs.com/#0JC454#16    remark-lint:double-link
  ✖  108:26  https://playground.babylonjs.com/#LU3FLF#6     remark-lint:double-link
  ✖  109:18  https://playground.babylonjs.com/#0IIB3N#44    remark-lint:double-link
  ✖  110:27  https://playground.babylonjs.com/#UFVU18#18    remark-lint:double-link
  ✖  111:[17](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:18)  https://playground.babylonjs.com/#ZS41RU#0     remark-lint:double-link
  ✖  112:21  https://playground.babylonjs.com/#K7TJIG       remark-lint:double-link
  ✖  113:[18](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:19)  https://playground.babylonjs.com/#6C8005#1     remark-lint:double-link
  ✖  114:24  https://playground.babylonjs.com/#IT1QZU#13    remark-lint:double-link
  ✖  115:18  https://playground.babylonjs.com/#YRP925       remark-lint:double-link
  ✖  116:18  https://playground.babylonjs.com/#3I55DK#0     remark-lint:double-link
  ✖  117:31  https://playground.babylonjs.com/#NE96B8#10    remark-lint:double-link
  ✖  118:22  https://playground.babylonjs.com/#MNC4HP       remark-lint:double-link
  ✖  1[19](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:20):19  https://playground.babylonjs.com/#8WC6ZN#112   remark-lint:double-link
  ✖  1[20](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:21):33  https://playground.babylonjs.com/#11XV5X#13    remark-lint:double-link
  ✖  1[21](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:22):37  https://playground.babylonjs.com/#4WPJCV#34    remark-lint:double-link
  ✖  1[22](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:23):31  https://playground.babylonjs.com/#ZENDZ2#0     remark-lint:double-link
  ✖  1[23](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:24):21  https://playground.babylonjs.com/#RYXIT#9      remark-lint:double-link
  ✖  1[24](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:25):26  https://playground.babylonjs.com/#2S8L5U#1     remark-lint:double-link
  ✖  1[25](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:26):32  https://playground.babylonjs.com/#ZU8CV4#12    remark-lint:double-link
  ✖  1[26](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:27):12  https://playground.babylonjs.com/#ACPMQL#2     remark-lint:double-link
  ✖  1[27](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:28):42  https://playground.babylonjs.com/#ARN6TJ#5     remark-lint:double-link
  ✖  1[28](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:29):20  https://playground.babylonjs.com/#1ND6TH#265   remark-lint:double-link
  ✖  1[29](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:30):19  https://playground.babylonjs.com/#1HH4OJ#29    remark-lint:double-link
  ✖  1[30](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:31):36  https://playground.babylonjs.com/#RBX8Y4#22    remark-lint:double-link
  ✖  1[31](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:32):44  https://playground.babylonjs.com/#1BGGDU       remark-lint:double-link
  ✖  1[32](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:33):26  https://playground.babylonjs.com/#S0IW99#23    remark-lint:double-link
  ✖  1[33](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:34):16  https://playground.babylonjs.com/#B2DP5I#7     remark-lint:double-link
  ✖  1[34](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:35):48  https://playground.babylonjs.com/#8WC6ZN#121   remark-lint:double-link
  ✖  1[35](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:36):38  https://playground.babylonjs.com/#10RCC9       remark-lint:double-link
  ✖  1[36](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:37):19  https://playground.babylonjs.com/#JXJBPS#5     remark-lint:double-link
  ✖  1[37](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:38):23  https://playground.babylonjs.com/#G03CV5#0     remark-lint:double-link
  ✖  1[38](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:39):46  https://playground.babylonjs.com/#0MGV2I#0     remark-lint:double-link
  ✖  1[39](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:40):19  https://playground.babylonjs.com/#ND4U2L       remark-lint:double-link
  ✖  1[40](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:41):31  https://playground.babylonjs.com/#2FPT1A#351   remark-lint:double-link
  ✖  1[41](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:42):18  https://playground.babylonjs.com/#Z88Q4W#19    remark-lint:double-link
  ✖  1[42](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:43):15  https://playground.babylonjs.com/#165IV6#2526  remark-lint:double-link
  ✖  1[43](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:44):20  https://playground.babylonjs.com/#1DDSV5       remark-lint:double-link
  ✖  1[44](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:45):32  https://playground.babylonjs.com/#52M011#2     remark-lint:double-link
  ✖  1[45](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:46):19  https://playground.babylonjs.com/#88MM6V#0     remark-lint:double-link
  ✖  1[46](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:47):22  https://playground.babylonjs.com/#CMS27E#11    remark-lint:double-link
  ✖  1[47](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:48):33  https://playground.babylonjs.com/#H19VH7#44    remark-lint:double-link
  ✖  1[48](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:49):19  https://playground.babylonjs.com/#XP6NNP#42    remark-lint:double-link
  ✖  1[49](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:50):21  https://playground.babylonjs.com/#C21DGD#3     remark-lint:double-link
  ✖  1[50](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:51):17  https://playground.babylonjs.com/#2WR12Z#2     remark-lint:double-link
  ✖  1[51](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:52):53  https://playground.babylonjs.com/#SQFG0Q#6     remark-lint:double-link
  ✖  1[52](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:53):[53](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:54)  https://playground.babylonjs.com/#SQFG0Q#5     remark-lint:double-link
  ✖  153:19  https://playground.babylonjs.com/#Z07JE1#2     remark-lint:double-link
  ✖  1[54](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:55):22  https://playground.babylonjs.com/#LPTLZM       remark-lint:double-link
  ✖  1[55](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:56):35  https://playground.babylonjs.com/#5W4GH1#10    remark-lint:double-link
  ✖  1[56](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:57):26  https://playground.babylonjs.com/#C6V6UY#49    remark-lint:double-link
  ✖  1[57](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:58):24  https://playground.babylonjs.com/#1JDJ9F#9     remark-lint:double-link
  ✖  1[58](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:59):20  https://playground.babylonjs.com/#9M1I08#7     remark-lint:double-link
  ✖  1[59](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:60):26  https://playground.babylonjs.com/#ZD364L#17    remark-lint:double-link
  ✖  1[60](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:61):31  https://playground.babylonjs.com/#7ILX7T       remark-lint:double-link
  ✖  1[61](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:62):16  https://playground.babylonjs.com/#WBEEFF#1     remark-lint:double-link
  ✖  1[62](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:63):25  https://playground.babylonjs.com/#PKQ6JV#3     remark-lint:double-link
  ✖  1[63](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:64):30  https://playground.babylonjs.com/#B922X8#19    remark-lint:double-link
  ✖  1[64](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:65):23  https://playground.babylonjs.com/#CHF3KW#108   remark-lint:double-link
  ✖  1[65](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:66):27  https://playground.babylonjs.com/#PPM311#99    remark-lint:double-link
  ✖  1[66](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:67):18  https://playground.babylonjs.com/#K1WGX0#36    remark-lint:double-link
  ✖  1[67](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:68):46  https://playground.babylonjs.com/#LYCSQ#256    remark-lint:double-link
  ✖  1[68](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:69):25  https://playground.babylonjs.com/#[73](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:74)TUXW#0     remark-lint:double-link

  [108](https://github.com/zakarialaoui10/linters/actions/runs/4164027614/jobs/7205128789#step:4:109) errors

Symbitic commented:
@zakarialaoui10 I already mentioned the errors above. Bug reports have been filed.

sindresorhus commented:
Please follow the normal formatting of Awesome lists. Links should be linkified text, not plain URL links.

sindresorhus commented:
> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

Symbitic commented:
> Please follow the normal formatting of Awesome lists. Links should be linkified text, not plain URL links.

@sindresorhus I tried that in the playground, but it doesn't fit with the `(bullet) name (dash) optional description (url)` format that the rest of the list uses. Is this really a requirement?

sindresorhus commented:
> Is this really a requirement?

Yes

Symbitic commented:
Very well. Changed the Playground format.

Symbitic commented:
@sindresorhus anything else needed? The latest version of remark-lint-double-link includes the fix for double hashtag links, and I'm trying to see about making a PR for awesome-lint to fix the heading issue. Can this be merged before awesome-lint is updated?

sindresorhus commented:
There are still many entries that have plain URLs.

For example:

> [InstancedBones](https://www.babylonjs.com/demos/instancedbones/) - A group of soldiers marching in sync. (https://github.com/BabylonJS/Website/tree/master/build/Demos/InstancedBones)

sindresorhus commented:
Bump

Symbitic commented:
I would like to request you add recommendations for what to do in this situation (entries with both a deployed URL and a source folder) to the official rules so that awesome-lint can just object to plain URLs.

Symbitic commented:
@sindresorhus All plain URLs have been removed.

sindresorhus commented:
`src` => `source`

`(src)` and `[demo]` are using inconsistent brackets

`Babylon.js Changelog`, `Babylon.js` is implied in this links and moot

sindresorhus commented:
`src` => `source`

`(src)` and `[demo]` are using inconsistent brackets

`Babylon.js Changelog`, `Babylon.js` is implied in this links and moot

Symbitic commented:
> `src` => `source`

Changed all instances of `src` to `source`.

> `(src)` and `[demo]` are using inconsistent brackets.

Switched to using square brackets for everything.

> `Babylon.js Changelog`, `Babylon.js` is implied in this links and moot

I removed `Babylon.js` where appropriate, but for some links, it is necessary because the link is specific to Babylon.js web; not Babylon Native (not the difference in spelling).

> Some items are missing a description.

I did my best to add a description to most links, but they are still missing in one area: the playgrounds.
It's hard to add descriptions to those because they are WYSIWYG. Most didn't have official titles so I had to come up with them myself, and none of them have official descriptions. Some are hard to describe in words, honestly.
What should I do here?

---

Thanks for bearing it out with me. I'm really hoping to finish this soon.

Symbitic commented:
@sindresorhus bump.

sindresorhus commented:
> Consistent and correct naming. For example, Node.js, not NodeJS or node.js.

BabylonJS and Babylon.js is used

> The description starts with an uppercase character and ends with a period.

Not all done.

sindresorhus commented:
> [Babylon JS Day 10: Terrarium](https://radicalappdev.com/2021/03/12/babylon-js-day-10-terrarium/) - Final in a series of lessons that explains how to use assetsManager.
[Babylon JS Day 19: Golf Club](https://radicalappdev.com/2021/03/25/babylon-js-day-19/) - Shows how to leverage AssetsManager to load a custom model as a VR controller.

Capitalization of `AssetManager`.

sindresorhus commented:
I would place the Playground section above Examples. It's very long and not as useful as many of the sections that come after it.

sindresorhus commented:
> BABYDOOM - Doom clone using the JavaScript Babylon.js engine. [demo]

Both "JavaScript" and "Babylon.js engine" are moot here as implied by the context. Same with many other entries.

Symbitic commented:
@sindresorhus The reason I used them is because those are from the official GitHub descriptions of the projects. Is it really okay to modify that?
